### PR TITLE
Set new (sensible) default values

### PR DESCRIPTION
**Pull request by aquamatthias**
--reconciliation_interval 600000
 --launch_tokens 100
 --max_tasks_per_offer 5

### CODE REVIEW

**Comment by kolloch** [comment]
The tests need to be adjusted.

**Comment by gkleiman** [comment]
@aquamatthias LGTM. Shouldn't we add a note in the change log?

**Comment by aquamatthias** [comment]
Good point. I added a section to the changelog.

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+
+### Overview
+
+#### Default settings for Task Launches have been changed
+
+Marathon has a lot of settings to adjust. Our goal is, to have sensible defaults for small and medium size clusters.
+We realized, that some default values are not sufficient in the field and changed them:
+- `--launch_tokens` has changed to 100 (was 1000)
+- `--max_tasks_per_offer` has changed to 5 (was 100) 
+- `--reconciliation_interval` has changed to 600000 (=10 minutes) (was 300000 (=5minutes))
```
Comment (gkleiman) [inline changelog.md]: s/5minutes/5 minutes/

**Comment by gkleiman** [comment]
LGTM! Please rebase so that I can hit the merge button =)

**Comment by gkleiman** [comment]
:rocket:

**Pull request merged**